docs/concepts/select.md

```diff
@@ -331,3 +294,85 @@ export class CityService {
 ```
 
 now we can use this `zooThemeParks` selector anywhere in our application.
+
+## Special Considerations
+
+### Angular Libraries: Use of lambdas in static functions
+
+_If you are building an Angular lib directly so that it can be deployed to npm. `strictMetadataEmit` will most likely be set to true in the Angular compiler options and, as a result, Angular's `MetadataCollector` from the `@angular/compiler-cli` package will report the following issue with using lambdas in static methods:_
+
+>  Metadata collected contains an error that will be reported at runtime: Lambda not supported.`
+
+This error would be reported for each of the selectors defined below but, as demonstrated in the sample, you can prevent this by including the `// @dynamic` comment before the class expression and decorators:
```

Comment:
Btw, I think this comment also supported when placed above the selector itself.

Reply:
Didn't seem to work in my tests. I read that it needs to be above an exported symbol. ie. above the declaration of something that has an associated `export`.